Tests/SPTPersistentCacheTests.m

```diff
@@ -1203,6 +1203,39 @@ - (void)testFailToRetrieveDirectoryContents
     XCTAssertTrue(called);
 }
 
+- (void)testNotFoundIfCacheDirectoryIsDeleted
+{
+    self.cache.workQueue = dispatch_get_main_queue();
+    [[NSFileManager defaultManager] removeItemAtPath:self.cachePath error:nil];
```

Comment:
Should we mock out NSFileManager perhaps? So that we don’t have to touch the file system. We could do that in a later PR if so.